packages/openactive-integration-tests/test/helpers/feature-helper.js

```diff
@@ -186,20 +197,36 @@ class FeatureHelper {
     ) {
       describe(configuration.testFeature, function () {
         describe(configuration.testIdentifier, function () {
-          if (configuration.runOnce) {
-            // This duplicate describe nesting ensures the number of describe levels remains consistent for the logger
-            describe(configuration.testIdentifier, function () {
-              const logger = new Logger(`${configuration.testFeature} >> ${configuration.testIdentifier}`, this, {
-                config: configuration,
-                description: configuration.testDescription,
-                implemented,
-              });
+          if (configuration.doesNotUseOpportunitiesMode) {
+            // Here we add some describe blocks so that these test uses the exact same number of describe levels as
+            // normal tests. This means that the logger (which looks at number of describe levels) can be used
+            // consistently across tests.
+            describe('_NoFlow_', () => {
+              describe('_NoOpportunityType_', function () {
+                const logger = new Logger(`${configuration.testFeature} >> ${configuration.testIdentifier}`, this, {
+                  config: configuration,
+                  description: configuration.testDescription,
+                  implemented,
+                });
 
-              const state = new RequestState(logger);
-              const flow = new FlowHelper(state);
+                const state = new RequestState(logger);
+                const flow = new FlowHelper(state);
 
-              tests.bind(this)(configuration, null, implemented, logger, state, flow);
+                tests.bind(this)(configuration, null, implemented, logger, state, flow);
+              });
             });
+            // describe(configuration.testIdentifier, function () {
+            //   const logger = new Logger(`${configuration.testFeature} >> ${configuration.testIdentifier}`, this, {
+            //     config: configuration,
+            //     description: configuration.testDescription,
+            //     implemented,
+            //   });
+
+            //   const state = new RequestState(logger);
+            //   const flow = new FlowHelper(state);
+
+            //   tests.bind(this)(configuration, null, implemented, logger, state, flow);
+            // });
```

Comment:
I've removed this on my local machine. I'll push once all the CI finishes. Otherwise, we'll have to wait again for it to complete 😢